packages/core/src/bus/Decorators/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Bind';
```

Comment:
The file doesn't export a class, don't need to use pascal case 

```suggestion
export * from './bind';
```